src/utils.py

```diff
@@ -39,11 +40,13 @@ def __init__(self, es_bucket=None, month=None, year=None):
             self.month, self.year = self.parse_date()
         else:
             self.y_axis = 0
+            nextDate = None
             if month < 10:
-                self.x_axis = f"01.0{month}.{year}"
+                nextDate = f"01.0{month}.{year}"
```

Comment:
Endre til next_date (hovedregel for variabelnavn i python er småbokstaver og understrek)